utilities/fzf/fzf.zsh

```diff
@@ -58,10 +58,22 @@ ctrl-u:preview-page-up,ctrl-a:select-all+accept"
 # << list files >>
 # NOTE: mapped to `C-f` instead of default `C-t`
 
-# paste selected files
-bindkey '^f' fzf-file-widget
-export FZF_CTRL_T_COMMAND="$FZF_DEFAULT_COMMAND"
```

Comment:
This might not really matter since we're aliasing shortcuts to it directly but I have a slight preference for setting this environment variable for `fzf` to reference